be/src/column/column_helper.cpp

```diff
@@ -95,6 +95,20 @@ void ColumnHelper::merge_two_filters(Column::Filter* __restrict filter, const ui
     }
 }
 
+void ColumnHelper::or_two_filters(Column::Filter* __restrict filter, const uint8_t* __restrict selected) {
+    uint8_t* data = filter->data();
+    size_t num_rows = filter->size();
+    for (size_t i = 0; i < num_rows; i++) {
+        data[i] |= selected[i];
+    }
+}
+
+void ColumnHelper::or_two_filters(size_t count, uint8_t* __restrict data, const uint8_t* __restrict selected) {
+    for (size_t i = 0; i < count; i++) {
+        data[i] |= selected[i];
+    }
+}
```

Comment:
Let the function above invoke the function below, and function below use simd or (_mm256_or_epi32)

Reply:
fixed

Reply:
We can add a SIMD::or function in simd.h file